test/suites/lxd_user.sh

```diff
@@ -38,3 +38,34 @@ test_lxd_user() {
 
   LXD_DIR="${bakLxdDir}"
 }
+
+snap_lxc_user() {
+    sudo -Hu testuser LXD_DIR=/var/snap/lxd/common/lxd-user lxc "${@}"
+}
+
+test_snap_lxd_user() {
```

Comment:
this test is a good example of a snap specific test, as it relies on the interplay between snap's systemd socket listeners and lxd-user inside the snap.